package.json

```diff
@@ -9,7 +9,7 @@
     "format-check": "prettier --check \"base/**/*.ts\"",
     "format": "npx prettier --write \"base/**/*.ts\"",
     "docs": "node scripts/generate_docs.js",
-    "postinstall": "husky install",
+    "_postinstall": "husky install",
```

Comment:
please revert this and let as is before you last commit. This is a repo bug that I still need to fix 😁

Reply:
I did that, but my prettier has added some whitespaces. it comes from the .prettier.json in the repo, which has tabwidth: 4 and i think when reverted the "_" the prettier worked according to the prettier config.

Reply:
This is an issue cloning or doing npm start... not sure. Just removing the `_` is fine :)